drivers/serial/uart_esp32.c

```diff
@@ -1029,7 +1031,9 @@ static const DRAM_ATTR struct uart_driver_api uart_esp32_api = {
 							     (UART_CFG_FLOW_CTRL_NONE)),           \
 						 COND_CODE_1(DT_INST_PROP(idx, hw_flow_control),   \
 							     (UART_CFG_FLOW_CTRL_RTS_CTS),         \
-							     (UART_CFG_FLOW_CTRL_NONE)))},         \
+							     (UART_CFG_FLOW_CTRL_NONE))),	\
+				.tx_invert = DT_INST_PROP_OR(idx, tx_invert, false),        \
+				.rx_invert = DT_INST_PROP_OR(idx, rx_invert, false)},         \
```

Comment:
`_OR` macro is not required for boolean
```suggestion
				.tx_invert = DT_INST_PROP(idx, tx_invert),        \
				.rx_invert = DT_INST_PROP(idx, rx_invert)},         \
```

Reply:
understood thanks!